integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateUtil.scala

```diff
@@ -166,127 +208,160 @@ object PreAggregateUtil {
       aggFunctions: AggregateFunction,
       parentTableName: String,
       parentDatabaseName: String,
-      parentTableId: String) : scala.collection.mutable.ListBuffer[(Field, DataMapField)] = {
+      parentTableId: String,
+      newColumnName: String) : scala.collection.mutable.ListBuffer[(Field, DataMapField)] = {
```

Comment:
what is this field? please add comment of this function

Reply:
fixed